src/playercards/CardsThatSealTokens.ttslua

```diff
@@ -289,20 +297,30 @@ function releaseAllTokens(playerColor)
   Player[playerColor].clearSelectedObjects()
 end
 
--- returns multiple tokens at once to the token pool
+-- returns multiple tokens at once to the token pool (with minimum)
 function returnMultipleTokens(playerColor)
   if SHOW_MULTI_RETURN <= #sealedTokens then
     for i = 1, SHOW_MULTI_RETURN do
       returnToken(table.remove(sealedTokens))
     end
     updateSave()
-    printToColor("Returning " .. SHOW_MULTI_RETURN .. " tokens", playerColor)
+    printToColor("Returning " .. SHOW_MULTI_RETURN .. " tokens to the token pool", playerColor)
   else
     printToColor("Not enough tokens sealed.", playerColor)
   end
   Player[playerColor].clearSelectedObjects()
 end
 
+-- returns all sealed tokens to the token pool
+function returnAllTokens(playerColor)
+  printToColor("Returning " .. #sealedTokens .. " tokens to the token pool", playerColor)
+  for i = 1, #sealedTokens do
+    returnToken(table.remove(sealedTokens))
+  end
+  updateSave()
+  Player[playerColor].clearSelectedObjects()
+end
```

Comment:
Can you explain why we need a new function (why is `returnMultipleTokens` not sufficient?)